tests/thread_stack_alignment/tests/01-run.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2022 Otto-von-Guericke-Universität Magdeburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+
+import sys
+import json
+from testrunner import run
+
+def testfunc(child):
```

Comment:
static tests is unhappy

```suggestion

def testfunc(child):
```

Reply:
Fixed, thanks. I also added the `README.md` for the test that I apparently forgot to write originally.